presets/laravel/config.yml

```diff
@@ -6,8 +6,8 @@ create:
   - name: Creating new Laravel Application
     actions:
       - scripts:
-          - docker pull -q kooldev/php:8.0
-          - kool docker kooldev/php:8.0 composer create-project --no-install --no-scripts --prefer-dist laravel/laravel $CREATE_DIRECTORY
+          - docker pull -q kooldev/php:8.1
```

Comment:
@fabriciojs - shouldn't we also update the Symfony preset to use PHP 8.1?

https://github.com/kool-dev/kool/blob/617017261e7662a2f52d4299eda7064ae540b3be/presets/symfony/config.yml#L9-L10

Reply:
@indiegenus yes, and no. This was not intended to be on this PR - will remove and we handle the change in another one.